v2/emailpassword/common-customizations/signup-form/field-validators.mdx

```diff
@@ -2,25 +2,40 @@
 id: field-validators
 title: Adding / Modifying field validators
 hide_title: true
+show_ui_switcher: true
 ---
 
-import FrontendSDKTabs from "/src/components/tabs/FrontendSDKTabs"
+import {PreBuiltOrCustomUISwitcher, PreBuiltUIContent, CustomUIContent} from "/src/components/preBuiltOrCustomUISwitcher"
+import FrontendPreBuiltUITabs from "/src/components/tabs/FrontendPreBuiltUITabs";
 import BackendSDKTabs from "/src/components/tabs/BackendSDKTabs"
 import TabItem from '@theme/TabItem';
 import CustomAdmonition from "/src/components/customAdmonition"
 
 # Adding / Modifying field validators 
 
-## Step 1: Front End
 
+<PreBuiltOrCustomUISwitcher>
+
+<CustomUIContent>      
+
+:::caution Not applicable
+This is not applicable in Custom UI, as you will be creating your own UI and can add field validators as needed. 
+However, it's crucial to ensure that these validations are also implemented on the backend. 
+:::
+
+</CustomUIContent>
```

Comment:
for this, for the frontend, step we can say that you need to implement this on your own. The backend step 2 and 3 will continue to be there for custom UI since it's needed there too.